core/cautils/datastructures.go

```diff
@@ -17,14 +17,15 @@ type KSResources map[string][]string
 type OPASessionObj struct {
 	K8SResources          *K8SResources                                 // input k8s objects
 	ArmoResource          *KSResources                                  // input ARMO objects
+	AllPolicies           *Policies                                     // list of all frameworks
```

Comment:
Can we remov `Policies`, `Exceptions`, and `RegoInputData`?

Reply:
why do we want to remove these?